containers/Ideas/IdeaCreate.js

```diff
@@ -134,13 +136,17 @@ export const IdeaCreate = props => {
               return `${acc}\n${curr}`
             }, 'Please try again.\n')
             setError(errorString)
+            setSubmitPending(false)
           } else {
             setError('Required fields are missing.')
+            setSubmitPending(false)
```

Comment:
if you want to have one line less you could shift it to the end of the statement, e.g. 
```
if (data && data.image) {
  ..
} else {
  ..
}
setSubmitPending(false)
```